Source/ProgressionSystemRuntime/Private/Components/PSSpotComponent.cpp

```diff
@@ -38,6 +42,13 @@ void UPSSpotComponent::BeginPlay()
 	Super::BeginPlay();
 	UPSWorldSubsystem& WorldSubsystem = UPSWorldSubsystem::Get();
 	WorldSubsystem.OnInitialize.AddDynamic(this, &ThisClass::OnInitialized);
+
+	// reset 
```

Comment:
Please explain the reset in comment for others, as this code might seem unusual to them without understanding the context. Example comment: 
```
//Before progression loaded, the game has all skins available by default.
// But if PS plugin is enabled, we want to change the default state with only the first skin unlocked.
// This should happen right after the progression module loads and before the saves are initialized.
```

Reply:
done https://github.com/moinm3uw/ProgressionSystem/pull/59/commits/8577657f7cdab104686de27e5fa354b318385f04